src/style.css

```diff
@@ -0,0 +1,128 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  padding: 32px 16px 0;
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__img {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 0 3px;
+  margin-bottom: 40px;
+}
+
+.img__imac {
+  width: 100%;
+  height: 134px;
+}
+
+.card__description {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  text-align: justify;
+}
+
+.card__product {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  padding-top: 4px;
+}
+
+.card__reviews {
+  display: flex;
+  padding: 16px 0;
+  justify-content: space-between;
+}
+
+.reviews__stars {
+  display: flex;
+}
+
+.stars__item {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-position: 50%;
+}
+
+.stars__item:last-child {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+}
```

Comment:
use `:nth-of-type()` instead, so the `stars--n` class will have sens in `.stars--n .stars__item:nth-of-type(n)`

Reply:
thanks